Gulpfile.js

```diff
@@ -10,9 +10,9 @@ var concat       = require('gulp-concat');
 var jsdoc        = require('gulp-jsdoc-to-markdown');
 var del          = require('del');
 var through      = require('through2');
-var Promise      = require('promise');
+var pify         = require('pify');
 
-var exec = Promise.denodeify(childProcess.exec);
+var exec = pify(childProcess.exec);
```

Comment:
This will not work in node 10: pify will fallback to the native promises https://github.com/sindresorhus/pify/blob/master/index.js#L37. Unfortunately we don't have tests for this. See how it's done in other packages.